src/WpfMath/Controls/FormulaControl.xaml.cs

```diff
@@ -175,8 +175,11 @@ private void Render()
 
             using (var drawingContext = visual.RenderOpen())
             {
-                texFormula.RenderTo(drawingContext, environment, Scale);
+                var renderer = new WpfElementRenderer(drawingContext, Scale);
+                renderer.RenderElement(formulaBox, 0, formulaBox.Height);
+                renderer.FinishRendering();
```

Comment:
Like it was done for the `TexFormula` type, this code can be moved to an extension method for the `Box` type, but I'm not sure this is of any interest.

Reply:
Meh, as long as it's only one place, that's fine to just write here.